src/Core/Components/Camera/CameraLensFlare.cpp

```diff
@@ -0,0 +1,95 @@
+#include "CameraLensFlare.h"
+#include "Utilities/Math/Math.h"
+#include "Core/Runtime/Reflection.h"
+
+namespace MxEngine
+{
+    float CameraLensFlare::GetLensFlareScale()const
+    {
+        return this->lensFlareScale;
+    }
+
+    float CameraLensFlare::GetLensFlareBias()const
+    {
+        return this->lensFlareBias;
+    }
+
+    int CameraLensFlare::GetLensFlareNumOfGhosts()const
+    {
+        return this->lensFlareNumOfGhosts;
+    }
+
+    float CameraLensFlare::GetLensFlareGhostDispersal()const
+    {
+        return this->lensFlareGhostDispersal;
+    }
+
+    float CameraLensFlare::GetLensFlareHaloWidth()const
+    {
+        return this->lensFalreHaloWidth;
+    }
+
+    void CameraLensFlare::SetLensFlareScale(float v)
```

Comment:
please give variables some meaningful name. I know it may feel kinda redundant as its setter, but I still suggest use smth like 'scale'